packages/react/src/components/Menu/_utils.js

```diff
@@ -135,7 +147,9 @@ export function getPosition(
   elementDimensions,
   targetBoundaries,
   containerBoundaries,
-  preferredDirection = 1
+  preferredDirection = 1,
+  isRootLevel,
+  element
```

Comment:
Looks great, thanks for the bug fix! Would you mind adding the two new parameters to the JSDoc annotation above?

Reply:
Forgot bout that, thanks 👍